apps/src/applab/constants.js

```diff
@@ -66,147 +66,118 @@ const THEME_ICON_BASE_URL = '/blockly/media/applab/theme_dropdown/';
 export const themeOptionsForSelect = [
   {
     option: 'default',
-    displayName: 'Default',
```

Comment:
Could you explain why we don't need `displayName`?

Reply:
We must use `I18n` functions, so there is no point to keep them

Reply:
I see how you updated `ThemeDropdown.jsx` to use the new `I18` functions 😄

Reply:
I see how you updated `ThemeDropdown.jsx` to use the new `I18` functions 😄